kafka-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/kafka/KafkaProperties.java

```diff
@@ -214,11 +214,24 @@ private static String resourceToPath(Resource resource) {
      */
     public static class Publisher {
 
+        /**
+         * Enables autoconfiguration for publishing events to Kafka
```

Comment:
Minor change suggestion in the description. Especially stating the default may be helpful to end-users.
```suggestion
         * Enables auto-configuration for publishing events to Kafka. Defaults to {@code true}.
```

Reply:
Applied this suggestion